spec/person_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rspec'
```

Comment:
- [ ] As required for the project to add unit tests for all classes that you created while **implementing the Decorator** design pattern, I think it would be better if you also implemented unit test for the missing classes and for all included methods as specified in the requirements.(e.g.: `Decorator`, `CapitalizeDecorator`, `TrimmerDecorator`, `Nameable`,...). Make sure you check all classes to be tested and for each one test its method(s). But Good job :100: :+1: